demos/3_Multichannel/03_Hyperspectral_reconstruction.ipynb

```diff
@@ -1326,9 +1290,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "recons = [FBP_recon_4D.get_slice(channel=20, vertical=35), cgls_reg_10.get_slice(channel=20, vertical=35),\\\n",
-    "          cgls_reg_10.get_slice(channel=20, vertical=35), PDHG_TV_1000_SpCh.get_slice(channel=20, vertical=35),\\\n",
-    "          FBP_recon_4D.get_slice(channel=20, horizontal_x=25), cgls_reg_10.get_slice(channel=20, horizontal_x=25),\\\n",
+    "recons = [FBP_recon_4D.get_slice(channel=20, vertical=35), cgls_reg_10.get_slice(channel=20, vertical=35),\n",
+    "          cgls_reg_10.get_slice(channel=20, vertical=35), PDHG_TV_1000_SpCh.get_slice(channel=20, vertical=35),\n",
+    "          FBP_recon_4D.get_slice(channel=20, horizontal_x=25), cgls_reg_10.get_slice(channel=20, horizontal_x=25),\n",
```

Comment:
here we are showing cgls_reg_10 twice in each row instead of showing cgls and cgls with tikhonov

Reply:
changing this is out of scope of this PR. Please open a separate issue/PR.